portal-frontend/src/app/services/notice-of-intent-submission/notice-of-intent-submission-draft.service.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { HttpClient } from '@angular/common/http';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { of, throwError } from 'rxjs';
+import { ToastService } from '../toast/toast.service';
+import { ApplicationSubmissionDraftService } from './application-submission-draft.service';
+
+describe('ApplicationSubmissionDraftService', () => {
+  let service: ApplicationSubmissionDraftService;
```

Comment:
Same as above re: intentional mention of ApplicationSubmissionDraftService in a notice-of-intent-submission-draft file

Reply:
File removed